src/web/app/pages-admin/admin-search-page/admin-search-page.component.ts

```diff
@@ -38,6 +38,8 @@ export class AdminSearchPageComponent {
   students: StudentAccountSearchResult[] = [];
   accountRequests: AccountRequestSearchResult[] = [];
 
+  readonly courseInRecycleBinTip = 'This course is in the recycle bin.';
```

Comment:
Not necessary to make this a field if it's only used once in the HTML file. Just write the whole string in the HTML file instead.

Reply:
Twice?